airbyte-config/config-persistence/src/test/java/io/airbyte/config/persistence/HealthCheckPersistenceTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.persistence;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.airbyte.db.Database;
+import java.sql.SQLException;
+import org.jooq.Result;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class HealthCheckPersistenceTest {
```

Comment:
Changes in this file:
* lift and shift of tests from `ConfigRepository`.
* intentionally not changing any of the tests themselves.